### ISSUE TITLE
None of the codesandbox examples work 

### ISSUE THREAD

**Issue by JonasGruenwald** [keystonejs/keystone#8462]
This applies to all of the examples, when clicking the Codesandbox link it fails to build with an error.
Error from [this example](https://github.com/keystonejs/keystone/tree/main/examples/usecase-blog) is the following:

```
yarn run v1.22.19
$ keystone dev
✨ Starting Keystone
✘ [ERROR] Could not resolve "../example-utils"

    keystone.ts:2:30:
      2 │ import { fixPrismaPath } from '../example-utils';
        ╵                               ~~~~~~~~~~~~~~~~~~

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentationabout this command.
```

Codesandbox link:
https://githubbox.com/keystonejs/keystone/tree/main/examples/usecase-blog

**Comment by husinzz** [comment]
in keystone.ts delete line 2, 11,12 in your sandbox, refresh the browser after the change

**Comment by Thinkscape** [comment]
Keystone is currently broken with latest version of next #8575

**Comment by kisankumavat** [comment]
@JonasGruenwald You have to copy this (https://github.com/keystonejs/keystone/blob/main/examples/example-utils.ts) file's content and create new file with the same name and put that file inside of root dir of that sandbox.

**Comment by sadakura** [comment]
Still not fixed :(

**Comment by dcousens** [comment]
@sadakura unfortunately we don't know of an easy fix for this at this time - open to any suggestions :)

**Comment by sadakura** [comment]
> @sadakura unfortunately we don't know of an easy fix for this at this time - open to any suggestions :)

sorry if I have insulted someone, no I'm not similar with these things to help, just looked up for demo :) great project anyway.

**Comment by JonasGruenwald** [comment]
The workaround suggested by @kisankumavat works fine btw.

I don't really care but I guess since `example-utils.ts` just exports a single line function it could just be added or in-lined into all the example repos?

**Comment by dcousens** [comment]
@kennedybaird indeed, the trade-off is helping make my life easier as a maintainer by ensuring the examples stay up to date.

I'm happy to inline the `fixPrismaPath` code, but the version updates are a pain and I'm trying to simplify our release process.
